imblearn/ensemble/easy_ensemble_generalization.py

```diff
@@ -0,0 +1,208 @@
+"Easy Ensemble Generalization"
+
+# Authors: Christos Aridas
+#
+# License: MIT
+
+import numpy as np
+
+from sklearn.base import ClassifierMixin, clone
+from sklearn.ensemble import BaseEnsemble, VotingClassifier
+from sklearn.tree import DecisionTreeClassifier
+from sklearn.utils import check_random_state
+from sklearn.utils.validation import check_is_fitted
+
+from ..pipeline import Pipeline
+from ..under_sampling import RandomUnderSampler as ROS
+
```

Comment:
I would use the full name since we are using it once :) Might be more intuitive and this is a burden to right it once.